### DISCUSSION TITLE
MTEX WORKSHOP 2023

### DISCUSSION THREAD

**Discussion by frankNiessen** [ORTools4MTEX/ORTools#24] [Announcements]
Dear ORTools and MTEX community,

we are very happy to announce the [MTEX workshop 2023](https://mtex-toolbox.github.io/workshop23) from **March 6th to March 15th 2023**. Although the workshop will allow for remote participation and all lectures will be streamed and recorded, we strongly encourage everybody to join in presence in order to enjoy the discussions, coffee breaks and excursion in real life.

The *first week* is an **MTEX training course** with focus on data cleaning, interpretation and visualization. Participants can send in their data, that will be analyzed during the exercises. The course also provides basic knowledge on crystallography and texture analysis, as we believe that MTEX significantly helps understanding these subjects. After successfully attending this course you will get a certificate which counts for 5 ETCS points.

The *second week* is planed as an **MTEX user meeting**. All current and future MTEX users are invited to share specific use cases, tips and tricks and discuss recent and upcoming developments in MTEX. The schedule includes introductory presentations of invited speakers to general use cases of MTEX as well as 20 minutes talks by the participants on specific applications. Another topic will be the substantial changes in MTEX 6.0 which will be released next year.

We are looking forward to a wonderful first workshop in presence after to years in the online word. Hope to see you soon.

Ralf Hielscher, Rüdiger Kilian, Luiz Morales and Frank Niessen.

**Comment by trudator** [comment]
Hi Frank, Thanks for sharing this. I will probably join and hope to see you there.
I don't see your presentation scheduled here btw?

**Reply by frankNiessen** [reply]
Hi, 

that is great to hear. Yes, I have been giving invited talks for the past two workshops so this year we should shift the focus a bit to make space for other topics than parent grain reconstruction as well. I will try to join but the date of the workshop conflicts with my parental leave, so I am not sure if I can join yet. If I'll be there I will of course give a talk. 

Best wishes
Frank

**Comment by jlhostis** [comment]
Hi ORTools team, 

This may not be the purpose of this thread, but I tried to register for the workshop on the MTEX page (before January 31st), and I saw they were having mail server issues so I used the provided gmail address (mtexworkshop@gmail.com), but got no response.

I'm a french PhD student and would like to follow the first week of the workshop remotely, maybe some of you know if the admissions are closed or if the issue is slowing things down ?

Thank you very much for your help,
Jules

**Reply by frankNiessen** [reply]
Hi, I will forward this to Ralf, I am sure he will get back to you.
